examples/plot_multi_axes.py

```diff
@@ -0,0 +1,121 @@
+"""
+Test Plots with Multiple Axes in Matplotcheck
+=============================================
+
+Below you will find some examples of how to use MatPlotCheck
+to test Matplotlib objects with more than one axes in Python.
+
+"""
+
+################################################################################
+# Import Packages
+# ---------------
+# To begin, import the libraries that you need.
+
+import matplotcheck.base as pt
+import matplotlib.pyplot as plt
+import pandas as pd
+
+###############################################################################
+# Setup
+# -----
+# First you must create some example data to plot over multiple axes. This
+# data could represent many different types of plots, such as plotting multiple
+# satellite bands, plotting NDVI of multiple study sites over time, and much
+# more.
+# Once you have created your plot, you will create a Matplotcheck
+# ``PlotTester`` object by providing each Matplotlib axes object to a unique
+# ``PlotTester`` object.
+
+# Create example data
+line_1 = pd.DataFrame({"Ax 1 X Vals": [(0), (10)], "Ax 1 Y Vals": [(0), (10)]})
+line_2 = pd.DataFrame({"Ax 2 X Vals": [(10), (0)], "Ax 2 Y Vals": [(0), (10)]})
+
+# Plot the example data on two seperate Axes
+fig, (ax1, ax2) = plt.subplots(1, 2, figsize = (10, 5))
+
+line_1.plot(ax=ax1, x="Ax 1 X Vals", y="Ax 1 Y Vals")
+line_2.plot(ax=ax2, x="Ax 2 X Vals", y="Ax 2 Y Vals")
+
+fig.suptitle("Figure Title")
+
+ax1.set(title="Axes 1", xlabel = "Ax 1 X Vals", ylabel = "Ax 1 Y Vals")
```

Comment:
Titles should be "Axis 1" and "Axis 2", as they are singular. ("axes" is for plural)

Reply:
I'm fairly sure it is "axes" here, as it's a class in [matplotlib](https://matplotlib.org/api/axes_api.html). Might be misunderstanding something though!

Reply:
Thanks @nkorinek for bringing this up!  I see what you mean by the class name. My understanding is that when you are referring to an individual plot, each plot is referred to as an individual "axis".  This applies to text only, not code, as in do you talk about a plot as an "Axes" or an "Axis".  

https://matplotlib.org/api/axis_api.html#matplotlib.axis.Axis
https://www.earthdatascience.org/courses/scientists-guide-to-plotting-data-in-python/plot-with-matplotlib/introduction-to-matplotlib-plots/

Since several of the open comments relate to this idea, so it would be good to have @lwasser weigh in on this.

Reply:
this is definitely confusing. We have axes objects and then subplots. i think the best language here is to stick with "figure" and subplots. Here is a matplotlib example and they use axis for each subplot title. i think there's no good way to avoid confusion here... so subplots may be best. You could label it subplot 1 (ax1) to avoid needing to use the word "axis" at all or "axes"

https://matplotlib.org/3.1.0/gallery/subplots_axes_and_figures/subplots_demo.html